File: NetFramework/WinForms/CS/TouchDemo/ZoomPages/Help UserControl/KeyboadControlBar.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace TouchToolKitDemo.C1ZoomPages.Help_Forms
{
    public partial class KeyboadControlBar : UserControl
    {
        public KeyboadControlBar()
        {
            InitializeComponent();
        }

        private void button_Click(object sender, EventArgs e)
        {
            SendKeys.Send((sender as Button).Tag.ToString());
        }

        Size? oldSize = null;
        private void button13_Click(object sender, EventArgs e)
        {
            if (oldSize == null)
            {
                oldSize = this.Size;
                this.Size = (sender as Control).Size;
            }
            else
            {
                this.Size = oldSize.Value;
                oldSize = null;
            }
        }
    }
}

File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/Help UserControl/KeyboadControlBar.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace C1ZoomPages.Help_Forms
	Public Partial Class KeyboadControlBar
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button_Click(sender As Object, e As EventArgs)
			SendKeys.Send(TryCast(sender, Button).Tag.ToString())
		End Sub

		Private oldSize As System.Nullable(Of Size) = Nothing
		Private Sub button13_Click(sender As Object, e As EventArgs)
			If oldSize Is Nothing Then
				oldSize = Me.Size
				Me.Size = TryCast(sender, Control).Size
			Else
				Me.Size = oldSize.Value
				oldSize = Nothing
			End If
		End Sub
	End Class
End Namespace
